README.md

```diff
@@ -362,13 +362,15 @@ Here is the overview of TiFlash architecture [The architecture of TiFlash's dist
 
 See [TiFlash Development Guide](/docs/DEVELOPMENT.md) and [TiFlash Design documents](/docs/design).
 
-Before submitting a pull request, please use [format-diff.py](format-diff.py) to format source code, otherwise CI build may raise error.
+Before submitting a pull request, please use [format-diff.py](format-diff.py) to format source code and resolve clang-tidy errors, otherwise CI build may raise error.
 
 > **NOTE**: It is required to use clang-format 12.0.0+.
 
 ```shell
 # In the TiFlash repository root:
```

Comment:
Maybe worth to mention that unlike clang-format, clang-tidy must be run after a compiling because it relies on the compile database.

Reply:
fixed!